PWGJE/Tasks/PhiInJetsJE.cxx

```diff
@@ -0,0 +1,412 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file PhiInJets.cxx
+/// \brief Reconstruction of Phi yield through track-track Minv correlations for resonance hadrochemistry analysis.
+///
+///
+/// \author Adrian Fereydon Nassirpour <adrian.fereydon.nassirpour@cern.ch>, Jimun Lee <jimun.lee@cern.ch>, Bong-Hwi Lim <bong-hwi.lim@cern.ch>, Sawan Sawan <sawan.sawan@cern.ch>
```

Comment:
Put one author per line.

Reply:
Fixed

Reply:
Why did you remove the other names? Are they not authors?